models/commentary.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Commentary extends Model {
+
+     static associate({ News,User  }) {
+      this.belongsTo(News, {
+        foreignKey: 'NewsId',
+        targetKey: 'id',
+      });
+      this.belongsTo(User, {
+        foreignKey: 'userId',
```

Comment:
This need to be changed to `UserId` according to `migrations/20211211155701-create-commentary.js`